From 4d2ed734f83ca718a04ccaff8acb8f81fd93c56d Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Mon, 2 Nov 2015 10:29:24 +0800
Subject: [PATCH] MA-7064 Ignore gcc4.9 Werror=date-time in rtl8821as

Ignore the compilation error caused by gcc4.9
Werror=date-time in rtl8821as driver.

Signed-off-by: Haoran Wang <b50027@freescale.com>
---
 drivers/net/wireless/rtlwifi/rtl8821as/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/net/wireless/rtlwifi/rtl8821as/Makefile b/drivers/net/wireless/rtlwifi/rtl8821as/Makefile
index 2e10540..0f39e50 100755
--- a/drivers/net/wireless/rtlwifi/rtl8821as/Makefile
+++ b/drivers/net/wireless/rtlwifi/rtl8821as/Makefile
@@ -14,7 +14,9 @@ EXTRA_CFLAGS += -Wno-unused-parameter
 EXTRA_CFLAGS += -Wno-unused-function
 EXTRA_CFLAGS += -Wno-unused
 #EXTRA_CFLAGS += -Wno-uninitialized
-#EXTRA_CFLAGS += -Wno-error=date-time	# Fix compile error on gcc 4.9 and later
+ifeq ($(shell expr `$(CC) -dumpversion | cut -f2 -d.` \>= 9), 1)
+EXTRA_CFLAGS += -Wno-error=date-time	# Fix compile error on gcc 4.9 and later
+endif
 
 EXTRA_CFLAGS += -I$(src)/include
 
-- 
1.8.0

